src/CoreBundle/Resources/public/javascript/table.js

```diff
@@ -117,4 +141,8 @@ $(document).ready(function () {
     });
 
     CHAMELEON.CORE.initializeEntryAutocomplete($('input#searchLookup'));
+
+    var $searchFields = $("input[name='_search_word']");
```

Comment:
we must only enable the event on the _search_word field of the list form. Either by limiting the selection to the correct form, or by adding a css class that marks the field clearly as being the search field. Otherwise we risk adding the event listener to fields for which it is not intended.

Reply:
Did that.

However the collision likelihood is now as big or bigger as this class name could also be used somewhere else.

Reply:
The idea was to use a css class that is unique for this purpose. But I think this is sufficiently unique